src/pages/Player.vue

```diff
@@ -629,6 +749,25 @@ export default {
         plyrProgressBar.appendChild(newMarker);
       });
     },
+    showScorecardMarkerOnSlider() {
+      // show the marker for scorecard on top of the video slider
+      var plyrProgressBar = document.querySelectorAll(".plyr__progress")[0];
+      let existingMarker = document.getElementById(`marker-scorecard`);
+      if (existingMarker != undefined) plyrProgressBar.removeChild(existingMarker);
```

Comment:
any reason ki existing wala hata rahe hain? agar existing hain toh phir there is nothing to do na?

Reply:
check if the function to render markers is being called multiple times